.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: Docker Build
+
+on: push
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Format Docker image name
+      run: echo IMAGE_REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
+    - name: Build a Docker image
+      run: docker build -t ${{ env.IMAGE_REPOSITORY }}:latest .
+    - name: Run Docker image
+      run: docker run --name mysql-db -d ${{ env.IMAGE_REPOSITORY }}:latest -e MYSQL_ROOT_PASSWORD=a029d0df84eb5549c641e04a9ef389e5
```

Comment:
🛑 **gitleaks** has detected a secret with rule-id `` in commit fac4ab999cdd91f8428b160ba3b19749cad4bbf5
If this secret is a true positive, please rotate the secret ASAP and rebase the commit containing the secret.